src/fiatExchanges/FiatExchangeCurrency.tsx

```diff
@@ -104,7 +107,10 @@ function FiatExchangeCurrency({ route, navigation }: Props) {
   const { flow } = route.params
 
   const [selectedCurrency, setSelectedCurrency] = useState<CiCoCurrency>(CiCoCurrency.cUSD)
-  const showEth = getFeatureGate(StatsigFeatureGates.SHOW_ETH_IN_CICO)
+  // TODO: Update this to actually respect all possible networkIds correctly
```

Comment:
is this in a follow up PR or is there a ticket for this?

Reply:
https://linear.app/valora/issue/ACT-938/update-cico-flow-to-correctly-respect-enabled-tokens-from-statsig